base/db/R/convert_input.R

```diff
@@ -0,0 +1,891 @@
+##' Convert between formats, reusing existing files where possible
```

Comment:
this looks like an existing file that should have been moved via `git mv` so as to retain version history

Reply:
@mdietze when I try to do it using `git mv` it says `fatal: not under version control, source=base/utils/R/convert_input.R, destination=base/db/R/convert_input.R`

Reply:
@nanu1605 make sure you use the right case in the paths.

Reply:
@mdietze Rebased to address this -- long story short it was less about `mv` vs add/`rm` and more that the old filename was retained with new content. Placing the defunct stub in a separate file allowed Git to detect the rename correctly.